simplq/src/components/pages/Status/QueueDetails.jsx

```diff
@@ -23,7 +26,7 @@ export default () => {
             <p>Creation time:</p>
           </tr>
           <tr>
-            <p>Total number of people in queue:</p>
+            <p>Total number joined so far in queue:</p>
```

Comment:
We have to hit a backend endpoint for getting this information:

```
GET /v1/queue/status?queueId={{queueId}}
```

It currently returns,
```
{
    "queueId": "81abb19b-c2c7-477d-aa53-efe501633743",
    "queueName": "NavaQ",
    "numberOfActiveTokens": 5,
    "totalNumberOfTokens": 6
}
```

So rather than passing the name down, let this component make the network call. We can then have this as an independent component, and use it here, as well as in the Joining page, or even in admin page or somewhere else later on.

Reply:
The date is not coming, I'll make the backend change, it should be easy.  https://github.com/SimplQ/simplQ-backend/issues/74

Reply:
Okay, so right now, we have the tokenId in the QueueStatus script and I need the queueId for this call. I have to use /token/${tokenId} right?

Reply:
oh I see. In `QueueStatus`, we should be having the `queueId` in `tokenStatusResponse`. It's not currently there. Let me make that change.

Reply:
I've made the change. queueId should be present in tokenStatusResponse